src/components/site/PageHeader.md

```diff
@@ -28,44 +28,55 @@ const breadCrumbs = () => (
   preTitle="Department Of"
   title="Public Works"
   deck="The Department of Public Works oversees and maintains the County’s public infrastructure, including highways, utilities, and trash and recycling."
-/>
+/>;
 ```
 
 Html Snippet:
 
 ```html
 <div class="dg_page-header blue">
-    <img class="dg_page-header__backgroundImage" src="http://baltimorecountymd.gov/sebin/j/x/public-works-header.jpg" aria-hidden="true">
-    <div class="dg_page-header__container">
-        <div class="container">
-        <div class="row">
-            <div class="col">
-            <ul class="dg_breadcrumbs">
-                <li class="dg_breadcrumb">
-                    <a title="Home" href="/">
-                        <i class="fas fa-home fa-1x" aria-hidden="true"></i><span class="sr-only">Home</span>
-                    </a>
-                </li>
-                <li class="dg_breadcrumb">
-                    <a title="This is a title" href="/departments">Departments</a>
-                </li>
-                <li class="dg_breadcrumb">
-                    <a title="This is a title" href="/departments/health">Department of Health</a>
-                </li>
-                <li>Adoptable Pets</li>
-            </ul>
-            </div>
+  <img
+    class="dg_page-header__backgroundImage"
+    src="http://baltimorecountymd.gov/sebin/j/x/public-works-header.jpg"
+    aria-hidden="true"
+  />
+  <div class="dg_page-header__container">
+    <div class="container">
+      <div class="row">
+        <div class="col">
+          <ul class="dg_breadcrumbs">
+            <li class="dg_breadcrumb">
```

Comment:
I think this maybe needs updated? I don't think `li` exists in the breadcrumbs anymore?

Reply:
That is correct. I overlooked the example in my original push. I'll update and re-commit.

Reply:
Update has been pushed with the most recent commit.